### ISSUE TITLE
Alarm is not working

### ISSUE THREAD

**Issue by verschoof** [BorisMorel/IcalBundle#7]
Hi,

I just tried out your code on the README:

```
$provider = $this->get('bomo_ical.ics_provider');

$tz = $provider->createTimezone();
$tz
    ->setTzid('Europe/Paris')
    ->setProperty('X-LIC-LOCATION', $tz->getTzid())
;

$cal = $provider->createCalendar($tz);

$cal
    ->setName('My cal1')
    ->setDescription('Foo')
;

$datetime = new \Datetime('now');
$event = $cal->newEvent();
$event
    ->setStartDate($datetime)
    ->setEndDate($datetime->modify('+5 hours'))
    ->setName('Event 1')
    ->setDescription('Desc for event')
    ->setAttendee('foo@bar.me')
    ->setAttendee('John Do')
;

$alarm = $event->newAlarm();
$alarm
    ->setAction('display')
    ->setDescription($event->getProperty('description'))
    ->setTrigger('-PT2H') //See Dateinterval string format
;

$calStr = $cal->returnCalendar();

return new Response(
    $calStr,
    200,
    array(
        'Content-Type' => 'text/calendar; charset=utf-8',
        'Content-Disposition' => 'attachment; filename="calendar.ics"',
    )
);
```

When I import this to my calendar (OSX iCal) then I dont see the given alarm... Is this a known issue?

**Comment by BorisMorel** [comment]
Hi,

Not a known issue.

Can you paste your ics generated file pls ?

**Comment by shokohsc** [comment]
Issue comes from Model\Alarm.php Line 27, replace 'display' to 'DISPLAY'.

**Comment by BorisMorel** [comment]
Fix with #11
